crates/fluvio-cli/src/client/home/status.rs

```diff
@@ -24,16 +24,18 @@ impl StatusOpt {
         let list = admin.all::<MirrorSpec>().await?;
         let now = SystemTime::now().duration_since(SystemTime::UNIX_EPOCH)?;
 
-        let outlist: Vec<(String, String, String, String)> = list
+        let outlist: Vec<(String, String, String, String, String, String)> = list
```

Comment:
this is bit too long of tuple.  should make it to struct

Reply:
Sure! Using a struct now!